service_listener.go

```diff
@@ -25,12 +26,25 @@ import (
 
 // Don't actually commit the changes to route53 records, just print out what we would have done.
 var dryRun bool
+// Sleep time in secs before checking
+var sleepTime int
 
 func init() {
 	dryRunStr := os.Getenv("DRY_RUN")
 	if dryRunStr != "" {
 		dryRun = true
 	}
+	
+	sleepTimeString := os.Getenv("SLEEP_TIME")
+	if (sleepTimeString != "") {
+		i64, err := strconv.ParseInt(sleepTimeString, 10, 32)
+		if (err != nil) {
+			fmt.Println("Error while trying to parse SLEEP_TIME env var")
```

Comment:
If we failed to parse this we would erroneously be setting the `sleepTime` to 0 and continuing on which might be unexpected. It might be more expected to exit with a non-zero exit code here as well as logging the above error.